### PR TITLE
Feature/drop down box on product name

### PR DESCRIPTION
**Pull request by LachlanMcCaskill**
The functionality appears to be working, but I don't feel satisfied with the way it looks right now. I would appreciate some feedback and suggestions.

### CODE REVIEW

**Comment by hd-chuong** [comment]
The autocomplete feature returns the correct suggestion. 

As product number is used to retrieve and edit details record, I think this features might need a function that maps the product names to its product number.

File: src/includes/getProductNames.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+$mysqli = new mysqli("mysql", "admin", "password", "SREPS");
+if($mysqli->connect_error) 
+{
+  exit('Could not connect');
+}
+
+$sql = "SELECT ProductName FROM Products";
```
Comment (hd-chuong) [inline src/includes/getProductNames.php]: Maybe add ProductNumber in this query.

**Comment by LachlanMcCaskill** [comment]
Once this branch has been reviewed and merged, please move it to "done" on trello if it hasn't already been moved by then.

**Comment by hd-chuong** [comment]
I am happy with this branch overall, given the fact that all the required feature has been met.

**Review by hd-chuong — APPROVED** [review]

**Pull request merged**